contextualized/modules.py

```diff
@@ -45,8 +45,12 @@ class MLP(nn.Module):
     """
     def __init__(self, input_dim, output_dim, width, layers, activation=nn.ReLU, link_fn=identity_link):
         super(MLP, self).__init__()
-        hidden_layers = lambda: [layer for _ in range(0, layers - 2) for layer in (nn.Linear(width, width), activation())]
-        mlp_layers = [nn.Linear(input_dim, width), activation()] + hidden_layers() + [nn.Linear(width, output_dim)]
+        if layers > 0:
+            mlp_layers = [nn.Linear(input_dim, width), activation()] + \
```

Comment:
Ben found before that the weight initializations for torch are slightly different (sqrt(1/n_features) vs 0.01)(https://pytorch.org/docs/stable/generated/torch.nn.Linear.html). Would we want to use this or separately initialize them?

Reply:
I think the default is okay for now (as long as we aren't mixing TF vs PyTorch, as you note), but we will want to allow custom inits in the future.

Reply:
Great comment! 

I agree with the new default. The only reason we would need the TF-Torch compatibility would be for reproducing the results in the NOTMAD preprint, but if we can reproduce those results with the new Torch version I think that may be less work. We can guarantee reproducibility within a torch version by using `torch.manual_seed(n)` and `numpy.random.seed(n)` prior to initializing a model and training. Between torch versions, we can use `contextualized.utils` to `save` and `load` weights. I can't think of a good reason to use custom weights otherwise, so seed-setting with model save/load load seems preferable, as long as we can reproduce the results in the NOTMAD pre-print. 

I can easily do this for the TCGA experiments once we merge the Torch NOTMAD #87 , @blengerich how difficult or neccessary do you think it is to reproduce the finance, sports, and SNARE-seq results in Torch?

Reply:
@cnellington We will want to reproduce the results using the Torch version, but this should be trivial now that @wtt102 's version is giving the same results as the TF version. Seeds will be important as well, I agree.

I believe this PR is ready to be merged as-is and we should make separate issues requesting addition of seed setting (maybe in `utils`) and custom inits.